horace_core/sqw/PixelData/@PixelDataMemory/apply_c.m

```diff
@@ -0,0 +1,27 @@
+function obj_out = apply_c(obj_in,page_op)
```

Comment:
at this point I believe we should have a document about how the PageOp class works, with examples and why it is a good idea. Happy to discuss where that should go, but for starters as an intro comment block for the pageopbase class would be ok

Reply:
I wrote Epic making the case for PageOp

Reply:
Then the `PageOp` contains all necessary comments.  Or at least I believe it does.